Muhasin/index.js

```diff
@@ -0,0 +1,35 @@
+// Variables
+let apples = 20;
+let oranges = 30;
+let mangos = 5;
+
+// Are apples and oranges equal?
+console.log(apples == oranges); // == false
+
+// Are apples and oranges unequal ?
+console.log(apples == oranges); // == true
+// are apples greater than oragnes ?
+console.log(apples > oranges); //false
+
+// are apples less than oragnes ?
```

Comment:
In exercise 4 you have to also check if apples are equal to oranges!!!
the solution is: 
console.log(apples <= oranges);